.circleci/config.yml

```diff
@@ -207,7 +207,9 @@ jobs:
           name: build all tools
           command: |
             cd cmds
-            go build -a -installsuffix uroot -ldflags '-s' ./...
+            go build -a ./...
```

Comment:
probably should be `go install` ?

Reply:
Just need to verify it builds - what does go install do that build doesn't?

Reply:
go build will generate a binary under the current directory, where go install will put the binary into `$GOPATH/bin`.

The circleci of the PR is failing because of `go build`:

```
go build github.com/u-root/u-root/tools/makebb: build output "makebb" already exists and is a directory
```

It will always succeed if `go install` is being used.